src/noos_inv/docker.py

```diff
@@ -52,13 +53,16 @@ def _dockerhub_login(ctx: Context, user: str, token: Optional[str]) -> None:
 
 
 @task
-def build(ctx, name=None, context=None, arg=None):
+def build(ctx, name=None, file=None, context=None, arg=None):
     """Build Docker image locally."""
     name = name or ctx.docker.name
+    file = file or ctx.docker.file
     context = context or ctx.docker.context
     arg = arg or ctx.docker.arg
     utils.check_path(context)
     cmd = f"docker build --pull --tag {name} "
```

Comment:
To add logic to check if the 'file' path exists in the file system (plus an additional small unit test?),

```python
utils.check_path(file)
```

Reply:
To get better clarity for distinguishing ill-formed function calls from failed underlying shell processes (not necessarily Python-based, thus uncertain about capturing correctly the stdout/stderr)?